### PR TITLE
Update Italian translation

### PR DESCRIPTION
**Pull request by Sav22999**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -2,70 +2,71 @@
...
 #: src/MainWindow.vala:54
 msgid "Dark background"
-msgstr ""
+msgstr "Sfondo scuro"
 
 #. TRANSLATORS: Whether to follow system's dark style settings
 #: src/MainWindow.vala:59
 msgid "Follow system style:"
-msgstr ""
+msgstr "Segui lo stile del sistema"
```
Comment (ryonakano) [inline po/it.po:65]:
```suggestion
msgstr "Segui lo stile del sistema:"
```

Seems like the trailing colon is missing, ignore my comment if it's intentional though.

**Review by ryonakano — COMMENTED** [review]
Hi thank you for your contribution! Just one question:

File: po/it.po (hunk 1 above)
Comment (Sav22999) [reply po/it.po:65]: Oh, yes, sorry. My mistake :smile:

**Review by ryonakano — APPROVED** [review]
LGTM, thanks!

**Pull request merged**